Pages/page2.html

```diff
@@ -0,0 +1,160 @@
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <div class="jumbotron">
+      <div class="container">
+        <h1>Attractions</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="Thumbnail col-md-4">
+        <h1>Sea Turtles</h1>
+        <video width="320" height="240" controls>
+          <source src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Videos/Seaturtle.mp4" type="video/mp4">
+        </video>
+      </div>
+
+      <div class="Thumbnail col-md-4">
+        <h1>Skydiving</h1>
+        <video width="320" height="240" controls>
+          <source src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Videos/Skydiving.mp4" type="video/mp4">
+        </video>
+      </div>
+
+      <div class="Thumbnail col-md-4">
+        <h1>Paragliding</h1>
+        <video width="320" height="240" controls>
+          <source src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Videos/Paragliding.mp4" type="video/mp4">
+        </video>
+      </div>
+    </div>
+
+    <div class="Container col-md-12">
+      <h1>Recommended Destinations</h1>
+      <div class="Table">
+        <table class="table table-bordered">
+          <thead>
+            <tr>
+              <th>Jan-March</th>
+              <th>April-June</th>
+              <th>July-Sept</th>
+              <th>Oct-Dec</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <tr>
+              <td>Yucatán</td>
+              <td>Greenville</td>
+              <td>Poland</td>
+              <td>Michigan</td>
+            </tr>
+
+            <tr>
+              <td>Quebec</td>
+              <td>Hawaii</td>
+              <td>Coasta Rica</td>
+              <td>Buenos Aires</td>
+            </tr>
+
+            <tr>
+              <td>Sonama, Calif</td>
+              <td>Lake Champlain</td>
+              <td>Chincoteague Island, Va</td>
+              <td>Newport, RI</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+
+    <div class="pagination">
+      <a href="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/index.html">&laquo;</a>
+      <a href="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/index.html">1</a>
+      <a href="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Pages/page2.html">2</a>
+      <a href="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Pages/page3.html">3</a>
+      <a href="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Pages/page3.html">&raquo;</a>
+    </div>
+
+  </body>
+
+
+  <footer>
+    <div class="footer col-md-12">
+      <div class="help col-md-4">
+        <label for="Help">Help</label>
+        <ul clas="Help">
+          <li><a href="#">Contact Us</a></li>
```

Comment:
I would avoid having dummy links like this that don't actually go anywhere in the final product.